pages/people_page.py

```diff
@@ -54,3 +56,30 @@ def we_filter_incorrect(self):
             print("\tData filtered correctly. Incorrect result was not displayed")
             self.context.wait.until(EC.element_to_be_clickable((By.XPATH, '//span[@title="Clear search"]')))
             self.context.driver.find_element_by_xpath('//span[@title="Clear search"]').click()
+
+    def create_list_name(self):
+        list_name = []
+        for i in self.context.driver.find_elements_by_css_selector('a.name'):
+            list_name.append(i.text)
+        return list_name
+
+    def set_list_name(self):
+        self.list_name = self.create_list_name()
```

Comment:
This look like redundant here, you can remove it and replace in step file "set_list_name" to "create_list_name" and nothing change.

Reply:
KISS principle (y)

Reply:
I use function "create_list name()" twice. Firstly to create list of all users and secondly to create "letter users". Function set_list_name() sets list of all users to list_name variable. The only solution may be to set the value directly in step realization, but it may be kind of logic in steps, which we are trying to avoid.